pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -27,7 +27,7 @@
 if _TORCH_GREATER_EQUAL_1_10:
     from torch import autocast
 else:
-    from torch.cuda.amp import autocast
+    from torch.cuda.amp import autocast  # type: ignore[misc]
```

Comment:
@carmocca here is an example of where mypy would not be aware of different torch versions

Reply:
Fixed

Reply:
You are too good at this ...